HLTrigger/Configuration/python/HLT_75e33/tasks/ecalDigisTask_cfi.py

```diff
@@ -1,7 +1,7 @@
 import FWCore.ParameterSet.Config as cms
 
```

Comment:
This file is still called `ecalDigisTask_cfi`, but now it defines the object `hltEcalDigisTask`. The file should be renamed `hltEcalDigisTask_cfi`, imho.